app/views/course/leaderboards/show.html.slim

```diff
@@ -1 +1,44 @@
-p To be implemented later
+= page_header
+
+div.col-sm-12.col-md-6
+  div.leaderboard-header
+    h4 = t('.level_header')
+  table.table.leaderboard-level
+    tbody
+      - @course_users.ordered_by_experience_points.take(display_user_count). \
+        each.with_index(1) do |course_user, index|
+        = content_tag_for(:tr, course_user, class: ['course-user'])
+          td.rank
+            h3 = index
+          td.user-picture
+            = link_to_course_user(course_user) do
+              = display_user_image(course_user.user)
+          td.user-profile
+            div
+              = link_to_course_user(course_user) do
+                = format_inline_text(course_user.name)
+            div
+              = t('.level', level_number: course_user.level_number)
+
+div.col-sm-12.col-md-6
+  div.leaderboard-header
+    h4 = t('.achievement_count_header')
+  table.table.leaderboard-achievement
+    tbody
+    - @course_users.ordered_by_achievement_count.take(display_user_count). \
```

Comment:
I believe there's no need for trailing \

In any case, in future it might be simpler to extract a variable first, then iterate over the variable.

```slim
- achievements = @course_users.ordered_by...take(display_user_count)
- achievements.each_with_index do...
```